job_creator.php

```diff
@@ -379,6 +387,13 @@ private function createPhpunitJobs(
                     'phpunit' => true,
                     'phpunit_suite' => $suite,
                 ]);
+                if ($this->getBranchVersion() >= '5.2') {
```

Comment:
This is a fragile check - we should be relying on the `INSTALLER_TO_PHP_VERSIONS` constant to tell us which PHP versions are allowed for which installer versions and to run a PHPUnit run for each PHP version supported for the installer version in question.

The only reason the current checks are there is because CMS 4 has different _databases_ it can run against (namely the PDO ones) - we shouldn't be using hardcoded conditions to determine which PHP jobs to run.

Reply:
Updated

Reply:
Nice - https://github.com/GuySartorelli/silverstripe-admin/actions/runs/7200793258 shows the new generated matrix for admin as an example.